File: threedollar-domain/src/main/java/com/depromeet/threedollar/domain/domain/admin/AdminCreator.java
package com.depromeet.threedollar.domain.domain.admin;

import com.depromeet.threedollar.common.docs.ObjectMother;
import lombok.AccessLevel;
import lombok.NoArgsConstructor;

@ObjectMother
@NoArgsConstructor(access = AccessLevel.PRIVATE)
public final class AdminCreator {

    public static Admin create(String email, String name) {
        return Admin.newInstance(email, name);
    }

}

File: threedollar-domain/src/testFixtures/java/com/depromeet/threedollar/domain/user/domain/admin/AdminCreator.kt
package com.depromeet.threedollar.domain.user.domain.admin

import com.depromeet.threedollar.domain.user.domain.TestFixture

@TestFixture
object AdminCreator {

    @JvmStatic
    fun create(
        email: String,
        name: String
    ): Admin {
        return Admin.builder()
            .email(email)
            .name(name)
            .build()
    }

}
